### ISSUE TITLE
(FR) Add another item to the Note Tmplate

### ISSUE THREAD

**Issue by davebhc71** [chhoumann/PodNotes#47]
Is there a chance you could provide as part of the Template, something similar to {{title}} or{{podcast}}, that would list the Sections/Chapters (e.g., {{chapters}} )within the Podcast?

**Comment by chhoumann** [comment]
Hi @davebhc71,

Thank you for the feature request.
I did some research on the feasibility of this.

Some notes:
- [PodcastIndex](https://podcastindex.org) API provides chapter URLs. However, doesn't seem to actually work from my testing (no chapter URLs, even on episodes with chapters).
- It might be possible to extract chapters from the audio file with `music-metadata-browser`, but can add a ton of overhead, impacting performance.
- From my testing, RSS feeds don't seem to contain chapters.

In short: could be possible, but would require a good bit of time to (potentially) do.
